src/css/_hero-stats.scss

```diff
@@ -119,9 +119,6 @@ hero-stats {
 		@include absolute(left 0 top 0);
 		@include size(100%);
 		z-index: -1;
-		@include ie {
```

Comment:
this was totally breaking the homepage background layout in IE causing the main H1 to be unreadable